hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -211,10 +235,48 @@ public Response getOpenKeyInfo(
         break;
       }
     }
+    // Populate the clusterSummary map
+    createClusterSummaryForOpenKey(clusterSummary);
+
+    openKeyInsightInfo.setClusterSummary(clusterSummary);
+
     openKeyInsightInfo.setLastKey(lastKey);
     return Response.ok(openKeyInsightInfo).build();
   }
 
+  private void createClusterSummaryForOpenKey(
+                Map<String, Object> clusterSummary) {
+    Long replicatedSizeOpenKey = getValueFromId(globalStatsDao.findById(
+        OmTableInsightTask.getReplicatedSizeKeyFromTable(OPEN_KEY_TABLE)));
+    Long replicatedSizeOpenFile = getValueFromId(globalStatsDao.findById(
+        OmTableInsightTask.getReplicatedSizeKeyFromTable(OPEN_FILE_TABLE)));
+    Long unreplicatedSizeOpenKey = getValueFromId(globalStatsDao.findById(
+        OmTableInsightTask.getUnReplicatedSizeKeyFromTable(OPEN_KEY_TABLE)));
+    Long unreplicatedSizeOpenFile = getValueFromId(globalStatsDao.findById(
+        OmTableInsightTask.getUnReplicatedSizeKeyFromTable(OPEN_FILE_TABLE)));
+    Long openKeyCountForKeyTable = getValueFromId(globalStatsDao.findById(
+        OmTableInsightTask.getTableCountKeyFromTable(OPEN_KEY_TABLE)));
+    Long openKeyCountForFileTable = getValueFromId(globalStatsDao.findById(
+        OmTableInsightTask.getTableCountKeyFromTable(OPEN_FILE_TABLE)));
+
+    // Calculate the total number of open keys
+    clusterSummary.put("totalOpenKeys",
+        openKeyCountForKeyTable + openKeyCountForFileTable);
+    // Calculate the total replicated and unreplicated sizes
+    clusterSummary.put("totalReplicatedDataSize",
```

Comment:
Pls Correct the javadoc. Still attributes name in response doesn't match.

Reply:
Corrected the doc! please check!